src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -158,4 +159,40 @@ Scalar<DataVector> area_element(
     const tnsr::i<DataVector, 3, Frame>& normal_one_form,
     const DataVector& radius,
     const tnsr::i<DataVector, 3, Frame>& r_hat) noexcept;
+
+/*!
+ * \ingroup SurfacesGroup
+ * \brief Spin function of a 2D `Strahlkorper`.
```

Comment:
Given that we might have multiple functions involving spin, I would suggest a name like CurvatureImaginaryPart (since the function parameters suggest we use this function with a surface)

Reply:
It is only proportional to the imaginary part of the complex curvature. "Spin function" is the name we used for this quantity for years in SpEC, so I'd rather not rename it now.